.github/workflows/lint.yml

```diff
@@ -26,7 +26,7 @@ jobs:
         python-version: 3.8
 
     - name: Install dependencies
-      run: pip install -U flake8 wheel setuptools yapf==0.31.0
+      run: pip install -U flake8 wheel setuptools yapf==0.31.0 toml
```

Comment:
In that case, it'll have to go [here](https://github.com/pyinstaller/pyinstaller/pull/6129/commits/4a7ba85a3049acf509ebd7afaca24fdfd58eb7a0#diff-a7d266f2057ae58d52a19900d891b63e200b5c4cef71053f6c4acc943fbb0962R17) too:

Reply:
Indeed.